server/urls.py

```diff
@@ -15,8 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path("", include("home.urls")),
     path("admin/", admin.site.urls),
-]
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Hmm, I'm pretty sure you don't need to manually add this... the static files middleware should do it automatically...

I'll try testing without it and push up the change...

I'll also add the whitenoise package to allow our python server to serve static assets in production, and we can put a free CloudFlare CDN in front of it so that they only need to be served up one before being cached by the CDN...

Reply:
Yup this is only for development.. Its usually hidden behind a if DEBUG flag but I forgot to put it in there.. 

And I think adding a hook to a CDN etc is an overkill since the static assets are extremely small and its also only for admin access which would be a limited group of people who wouldn't mind waiting a few milliseconds..